diffkemp/simpll/passes/SimplifyKernelFunctionCallsPass.cpp

```diff
@@ -85,10 +87,7 @@ PreservedAnalyses SimplifyKernelFunctionCallsPass::run(
                     newCall->setDebugLoc(CallInstr->getDebugLoc());
                     CallInstr->replaceAllUsesWith(newCall);
                     toRemove.push_back(&Instr);
-                } else if (CalledFun->getName() == "_dev_info" ||
-                           CalledFun->getName() == "dev_warn" ||
-                           CalledFun->getName() == "dev_err" ||
-                           CalledFun->getName() == "sprintf") {
+                } else if (isPrintFunction(CalledFun->getName())) {
```

Comment:
I think we do, because although the functions are considered equal, the parameters (which are compared separately from the function) are not. (Previously probably all functions that had the same parameters were compared as equal because of #85.)

Reply:
Agreed, let's keep it there.